test/data/testvib.py

```diff
@@ -77,6 +106,9 @@ def testzeropointcorrection(self):
 class JaguarIRTest(GenericIRTest):
     """Customized vibrational frequency unittest"""
 
+    max_force_constant = 3.7
+    max_reduced_mass = 2.3

```

Comment:
Can you explain why this is necessary? Is the geometry different from other packages?

Reply:
I manually checked Jaguar IR test data (`data/Jaguar/basicJaguar7.0/dvb_ir.out`), and I found that the maximum value of force constant and reduced mass are 3.70 mDyne/angstrom and 2.31 dyne.
Gaussian, QChem and other simulation packages outputs vibrational inf with normal coordinates, but Jaguar outputs with cartesian coordinates. It would be the reason that Jagur output diffretent values for reduced masses and force constants. I checked that vibrational displacements of Jaguar  are also differ from those of Gaussian or QChem.